lib/util/blob.js

```diff
@@ -67,4 +68,13 @@ async function blobResponse(s3, filename, blob) {
   }
 }
 
-module.exports = { blobContent, blobResponse, streamBlobs, streamEncBlobs };
+function defaultMimetypeFor(filename) {
+  if (!filename) return null;
+  switch (extname(filename).toLowerCase()) {
+    case '.csv':     return 'text/csv'; // eslint-disable-line no-multi-spaces
+    case '.geojson': return 'application/geo+json';
+    default:         return null; // eslint-disable-line no-multi-spaces
```

Comment:
When combined with the other PR to not allow null content types in the database, this part will still return null but then the DB will provide the default value, yes?

Reply:
Now that https://github.com/getodk/central-backend/pull/1355 has been merged, this may no longer be necessary - I will confirm.

Reply:
Correct!